guides/source/getting_started.md

```diff
@@ -1861,9 +1861,11 @@ Our blog has <%= Article.public_count %> articles and counting!
 
 <ul>
   <% @articles.each do |article| %>
-    <li>
-      <%= link_to article.title, article %>
-    </li>
+    <% unless article.archived? %>
```

Comment:
An early code block in [Section 8.3](https://guides.rubyonrails.org/getting_started.html#using-concerns) showed an `unless` block:

```erb
<h1>Articles</h1>

<ul>
  <% @articles.each do |article| %>
    <% unless article.archived? %>
      <li>
        <%= link_to article.title, article %>
      </li>
    <% end %>
  <% end %>
</ul>

<%= link_to "New Article", new_article_path %>
```

The `app/views/articles/index.html.erb` file isn't mentioned after here, so I think we could add the `unless` block  here too? Otherwise, a developer may get into an incorrect state, such as one shown in the screenshot below:

<img width="280" alt="screenshot" src="https://user-images.githubusercontent.com/16869656/105184037-2c509080-5b2f-11eb-9338-e8f9de72be95.png">